editParagraph.php

```diff
@@ -0,0 +1,44 @@
+<?php
+	/**
+	 * Created by PhpStorm.
+	 * User: academy
+	 * Date: 2019-03-15
+	 * Time: 16:26
+	 */
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<title>Jamila Ferron | Dashboard</title>
+	<link type="text/css" rel="stylesheet" href="css/normalize.css">
+	<link href="https://fonts.googleapis.com/css?family=Coiny|Covered+By+Your+Grace|Muli" rel="stylesheet">
+	<link type="text/css" rel="stylesheet" href="css/style.css">
+	<link type="text/css" rel="stylesheet" href="css/cms.css">
+</head>
+<body>
+	<header class="container header-section">
+		<nav>
+			<img src="img/bars.svg" alt=""/>
+			<ul>
+				<li><a href="#">Log Out</a></li>
+			</ul>
+			<h1>Dashboard</h1>
+		</nav>
+	</header>
+	<main>
+		<section class="add-paragraph">
+			<h1>Edit Paragraph</h1>
+			<form class="add-form">
+				<textarea rows="20" cols="63">In my spare time I enjoy learning new creative skills which require me to use my hands and combine multiple ideas to form new creations. In recent years I have taught myself how to sew, knit and crochet. When I am not working or indulging in my hobbies you will find me providing support to young people in semi independence accommodation.
+				</textarea>
+				<input type="submit" name="edit_submit">
+			</form>
+		</section>
+		<a href="dashboard.php"><button class="back-button">Back</button></a>
```

Comment:
You shouldn't put a button inside a link. A block level element should not go into an inline-level element.